ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 65.43 s <<< FAILURE! -- in osplus.pkptuna.hazelcast.pubsub.GenericHazelcastConsumerTest
[ERROR] osplus.pkptuna.hazelcast.pubsub.GenericHazelcastConsumerTest.testPollRequestMaps_PositiveFlow -- Time elapsed: 13.31 s <<< ERROR!
org.awaitility.core.ConditionTimeoutException: Condition with Lambda expression in osplus.pkptuna.hazelcast.pubsub.GenericHazelcastConsumerTest was not fulfilled within 1 seconds.
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.awaitility.core.CallableCondition.await(CallableCondition.java:78)
	at org.awaitility.core.CallableCondition.await(CallableCondition.java:26)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:975)
	at osplus.pkptuna.hazelcast.pubsub.GenericHazelcastConsumerTest.setUp(GenericHazelcastConsumerTest.java:49)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:101)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:81)
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:103)
	... 8 more

----


2025-07-16T18:32:10.478+02:00  INFO 24532 --- [uui] [ached.thread-19] c.h.t.TransactionManagerService          : [16.160.188.219]:5704 [test-cluster] [5.3.0] Committing/rolling-back live transactions of [16.160.188.219]:5701, UUID: bebe5a0c-5687-4e4e-919e-b2e275099a1d
2025-07-16T18:32:10.481+02:00  WARN 24532 --- [uui] [.async.thread-2] c.h.s.i.o.OperationService               : [16.160.188.219]:5704 [test-cluster] [5.3.0] Member [16.160.188.219]:5701 - bebe5a0c-5687-4e4e-919e-b2e275099a1d has left cluster!
2025-07-16T18:32:10.482+02:00  WARN 24532 --- [uui] [.async.thread-2] c.h.s.i.o.OperationService               : [16.160.188.219]:5704 [test-cluster] [5.3.0] Member [16.160.188.219]:5701 - bebe5a0c-5687-4e4e-919e-b2e275099a1d has left cluster!
2025-07-16T18:32:10.591+02:00  INFO 24532 --- [uui] [patia.migration] c.h.i.partition.impl.MigrationManager    : [16.160.188.219]:5704 [test-cluster] [5.3.0] Repartitioning cluster data. Migration tasks count: 179
2025-07-16T18:32:10.768+02:00  INFO 24532 --- [uui] [patia.migration] c.h.i.partition.impl.MigrationManager    : [16.160.188.219]:5704 [test-cluster] [5.3.0] All migration tasks have been completed. (repartitionTime=Wed Jul 16 18:32:10 CEST 2025, plannedMigrations=179, completedMigrations=179, remainingMigrations=0, totalCompletedMigrations=1589)
2025-07-16T18:32:10.946+02:00 ERROR 24532 --- [uui] [           main] o.p.h.pubsub.GenericHazelcastConsumer    : Error processing request from map-0 for flow 'myFlow': Expected failure

java.lang.RuntimeException: Expected failure
	at osplus.pkptuna.hazelcast.test.config.FlowHandlerTestConfiguration$1.handleRequest(FlowHandlerTestConfiguration.java:96) ~[test-classes/:na]
	at osplus.pkptuna.hazelcast.pubsub.GenericHazelcastConsumer.processEntry(GenericHazelcastConsumer.java:98) ~[classes/:na]
	at osplus.pkptuna.hazelcast.pubsub.GenericHazelcastConsumer.pollRequestMaps(GenericHazelcastConsumer.java:76) ~[classes/:na]
	at osplus.pkptuna.hazelcast.pubsub.GenericHazelcastConsumerTest.testPollRequestMaps_HandlerThrowsException(GenericHazelcastConsumerTest.java:78) ~[test-classes/:na]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[na:na]
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:775) ~[junit-platform-commons-1.12.2.jar:1.12.2]
	at org.junit.platform.commons.support.ReflectionSupport.invokeMethod(ReflectionSupport.java:479) ~[junit-platform-commons-1.12.2.jar:1.12.2]
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:161) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:152) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:91) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:112) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:94) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64) ~[junit-jupiter-engine-5.12.2.jar:5.12.2]
